File: Python/1_Iniciante/1046.py
ipt = [int(i) for i in input().split()]
a = ipt[0]
b = ipt[1]
if(a>=b):
    print(f'O JOGO DUROU {(24-a+b)} HORA(S)')
else:
    print(f'O JOGO DUROU {(b-a)} HORA(S)')

File: Ruby/1_Iniciante/1046.rb
ipt = gets.split().map(&:to_i)
a = ipt[0]
b = ipt[1]
if a >= b
    puts "O JOGO DUROU #{(24-a+b)} HORA(S)"
else
    puts "O JOGO DUROU #{(b-a)} HORA(S)"
end
